config/config.go

```diff
@@ -124,15 +142,23 @@ func (c *Config) Ensure(fromCloud bool, logger golog.Logger) error {
 		} else {
 			c.Components[idx].ImplicitDependsOn = dependsOn
 		}
+		if err := c.validateUniqueResource(seenResources, c.Components[idx].ResourceName().String()); err != nil {
+			logger.Error(err.Error())
+		}
 	}
 
 	for idx := 0; idx < len(c.Processes); idx++ {
+		// for processes it has to be local -- but still dependent on the API
```

Comment:
I don't understand this comment either.

Reply:
ah these are all my comments will remove haha